VCEntities/VCEntities/issuance/IssuanceResponseContainer.swift

```diff
@@ -11,26 +11,21 @@ public struct IssuanceResponseContainer: ResponseContaining {
     let expiryInSeconds: Int
     public let audienceUrl: String
     public let audienceDid: String
-    public var issuancePin: String? = nil
+    public var issuancePin: IssuancePin? = nil
     public var issuanceIdToken: String? = nil
     public var requestedIdTokenMap: RequestedIdTokenMap = [:]
     public var requestedSelfAttestedClaimMap: RequestedSelfAttestedClaimMap = [:]
     public var requestVCMap: RequestedVerifiableCredentialMap = [:]
     
     public init(from contract: Contract,
                 contractUri: String,
-                expiryInSeconds exp: Int = 3000,
-                rawIssuerIdToken: String? = nil,
```

Comment:
There is no change in idtoken being sent. Was this change intentional?

Reply:
yep, its an optimization that I saw and wanted to fix